internal/install/command.go

```diff
@@ -98,7 +98,8 @@ func assertProfileIsValid() error {
 		return fmt.Errorf("region is required")
 	}
 
-	licenseKey, err := client.FetchLicenseKey(accountID, config.FlagProfileName)
+	maxRetries := 60
+	licenseKey, err := client.FetchLicenseKey(accountID, config.FlagProfileName, &maxRetries)
```

Comment:
Something to consider: this assumes a global default retry delay of 5s. If the default retry delay changes, the client will not be retrying for 5 minutes. If we want to guarantee 5 minutes, we could either:
1. Set max retries and delay explicitly (`maxRetries = 60, delay = 5000`)
2. Calculate max retries backwards from 5 minutes `maxRetries = 300_000 / delay`

Reply:
Fair point, I'll see what adjustments I can make.